### PR TITLE
Eng 10147 Stream view showcase review

### PR DESCRIPTION
**Pull request by zeemanhe**

### CODE REVIEW

File: db/procedures/events/GetTopSources.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+public class GetTopSources extends VoltProcedure {
+    final SQLStmt getTopSource = new SQLStmt(
+        "SELECT src as Sources, total_visits as Counts  FROM events_by_src_view ORDER BY total_visits DESC LIMIT ?;"
+    );
+
+    public VoltTable run(int n) throws UnknownHostException
+    {
+        voltQueueSQL(getTopSource, n);
+        final VoltTable result = voltExecuteSQL(true)[0];
+        final VoltTable.ColumnInfo[] schema = result.getTableSchema();
```
Comment (akhanzode) [inline db/procedures/events/GetTopSources.java]: Why do you need to look at schema? You can simply return what you get from query right?

File: db/procedures/events/GetTopSrcDests.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    		"SELECT src as Sources, url as Destination, counts  FROM events_by_src_dest_view, dests "+
+    		"WHERE dest = dests.id ORDER BY counts DESC LIMIT ?;"
+    );
+
+    public VoltTable run(int n) throws UnknownHostException
+    {
+        voltQueueSQL(getTopSrcSource, n);
+        final VoltTable result = voltExecuteSQL(true)[0];
+        final VoltTable.ColumnInfo[] schema = result.getTableSchema();
+        schema[0] = new VoltTable.ColumnInfo("SOURCE", VoltType.STRING);
```
Comment (akhanzode) [inline db/procedures/events/GetTopSrcDests.java]: same as above comment.

Comment (zeemanhe) [inline db/procedures/events/GetTopSrcDests.java]: The sources in the query results are converted to IP address before returning. The schema is used to instantiate VoltTable.

File: run.sh [hunk 3]
```diff
@@ -224,7 +220,7 @@ function hadoop_demo() {
 }
 
 function help() {
-    echo "Usage: ./run.sh {clean|catalog|demo|hadoop_demo|start_web|stop_web}"
+    echo "Usage: ./run.sh {clean|d|demo|hadoop_demo|start_web|stop_web}"
```
Comment (vtkstef) [inline run.sh]: a single `d` for what?

File: db/procedures/events/GetTopSrcDests.java (hunk 2 above)
Comment (akhanzode) [inline db/procedures/events/GetTopSrcDests.java]: Do you need them as IP address in javascript? Can you not deal with them as string in javascript?

File: run.sh (hunk 3 above)
Comment (zeemanhe) [inline run.sh]: revert it back to catalog.

File: db/procedures/events/GetTopSrcDests.java (hunk 2 above)
Comment (zeemanhe) [inline db/procedures/events/GetTopSrcDests.java]: The src is stored as integer in voltdb.  The javascript shows what the query returns. The integers may not mean much to the users. Thus ip lookup is done before the query returns.

Comment (zeemanhe) [inline db/procedures/events/GetTopSrcDests.java]: moved the ip conversion to client and eliminated the schema lookup in all  stored procedures.

**Pull request merged**